docs/app/pages/components/components-sections/tables/sorting/sorting.component.html

```diff
@@ -1,17 +1,30 @@
 <div class="sorter-container">
     <span class="sorter-title">Sort by</span>
 
-    <div class="btn-group filter sort-direction-toggle" dropdown>
+    <div class="btn-group filter sort-direction-toggle"
+        uxMenuNavigation
+        (navigatedOut)="toggle.focus()"
+        [toggleButton]="toggle"
+        dropdown
+        #dropdown="bs-dropdown">
 
         <!-- Dropdown Button -->
-        <button dropdownToggle class="btn dropdown-toggle filter-dropdown">
+        <button #toggle="uxMenuNavigationToggle"
+            uxMenuNavigationToggle
+            [(menuOpen)]="dropdown.isOpen"
+            dropdownToggle
+            class="btn dropdown-toggle filter-dropdown">
             {{ selected }} <span class="hpe-icon hpe-down"></span>
         </button>
 
         <!-- Dropdown List -->
         <ul *dropdownMenu class="dropdown-menu" role="menu">
             <li role="menuitem" *ngFor="let option of options">
```

Comment:
Also here I think `role="menuitem"` needs to be on the focusable element which presumably is the `a`.